client/login/wp-login/index.jsx

```diff
@@ -361,13 +380,17 @@ export class Login extends Component {
 			</Main>
 		);
 
+		const shouldHideCreateAccountLink =
+			action === 'lostpassword' || action === 'jetpack/lostpassword';
```

Comment:
Alternatively, we can move this code within `OneLoginLayout` component

Reply:
Thanks. In https://github.com/Automattic/wp-calypso/pull/104948 , I'm also passing a `isSectionSignup` prop. We could also pass something like `isLostPassowordView`? Like we have for the one-login-footer the `isLoginView`? 🤔